### PR TITLE
Apply PEP 8 conventions and optimize code

### PR DESCRIPTION
**Pull request by MunawarAliArbisoft**
Apply pep8 conventions and optimize the code.

### CODE REVIEW

File: typing_tester.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+from curses import wrapper
+import time
+
+def cal_accuracy_score(given_word, user_text, no_of_backspaces):
+    """Calculates the accuracy score for typing tester"""
+
+    wrong_letter = 0
+    try:
+        wrong_letter = sum([1 for i in range(len(given_word)) if user_text[i] != given_word[i]])
+    except:
```
Comment (Itezaz-ul-Hassan) [inline typing_tester.py:11]: Its recommended to catch specific type of exception for better readability, logic and it allows us to handle specific exception cases

Comment (Itezaz-ul-Hassan) [inline typing_tester.py:11]: Its recommended to catch specific type of exception for better readability, logic and it allows us to handle specific exception cases

File: typing_tester.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        wrong_letter += 1
+
+    total_mistakes = wrong_letter + no_of_backspaces
+    accuracy = len(given_word) - total_mistakes
+    accuracy =  accuracy/len(given_word)*100
```
Comment (Itezaz-ul-Hassan) [inline typing_tester.py:16]: Whats the difference between the accuracy on line 15 and 16? Always use meaningful names for variables

File: typing_tester.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+
+def cal_time_score(elapsed_time, given_word):
+    """Calculates the time score for typing tester"""
+
+    # Avg. time for one keystroke
+    keystroke_time = 0.28 
+    # Avg. time to hover hands from mouse to keyboard
+    mouse_to_keyboard_time = 0.4 
+    # formula to calculate raw score
```
Comment (Itezaz-ul-Hassan) [inline typing_tester.py:28]: Always use all CAPS for constants

File: typing_tester.py [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    raw_score = elapsed_time / (mouse_to_keyboard_time + (keystroke_time * (len(given_word) + 1)))
+
+    if raw_score < 1:
+         return float(100)
+    else:
+         return float(f"{(100/raw_score):.2f}")
```
Comment (Itezaz-ul-Hassan) [inline typing_tester.py:34]: Make use of ternary operator to make it one liner

File: typing_tester.py [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    """Displays the user input dynamically on each keystroke"""
+
+    stdscr.addstr(2, 0, "||*** Typing Tester ***||", curses.color_pair(1))
+    stdscr.addstr(4, 0, "Press Enter when done...")
+    stdscr.addstr(6, 0, f"Word : {word}")
+    stdscr.addstr(7, 0, "Enter the above word: ")
+
+    # for loop is used for every key user input
+    for char in user_text:
+        stdscr.addstr(char, curses.color_pair(1))
```
Comment (Itezaz-ul-Hassan) [inline typing_tester.py:90]: Make it one liner

File: typing_tester.py [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+        # display user text everytime user enters key
+        display_text(stdscr, given_word, user_text) 
+        stdscr.refresh()
+        key = stdscr.getkey()
+
+        if ord(key) == DELETE_KEY_ASCII:
+            if len(user_text) > 0:
+                    user_text.pop()
+                    backspaces = backspaces + 1
```
Comment (Itezaz-ul-Hassan) [inline typing_tester.py:114]:
```suggestion
                    backspaces += 1
```

**Review by Itezaz-ul-Hassan — CHANGES_REQUESTED** [review]
Now divide the code in multiple modules

**Comment by Itezaz-ul-Hassan** [comment]
Always provide description in PR, what's the change, why was it needed and so on.

File: typing_tester.py (hunk 4 above)
Comment (MunawarAliArbisoft) [reply typing_tester.py:34]: Resolved

File: typing_tester.py (hunk 5 above)
Comment (MunawarAliArbisoft) [reply typing_tester.py:90]: Resolved

File: typing_tester.py (hunk 6 above)
Comment (MunawarAliArbisoft) [reply typing_tester.py:114]: Resolved

File: typing_tester.py (hunk 3 above)
Comment (MunawarAliArbisoft) [reply typing_tester.py:28]: Resolved

**Comment by MunawarAliArbisoft** [comment]
> _ No description provided. _

File: typing_tester.py (hunk 1 above)
Comment (MunawarAliArbisoft) [reply typing_tester.py:11]: Resolved

**Pull request closed without merging**